src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -767,6 +766,16 @@ private async Task AddDocumentsWithPotentialConflictsAsync(IEnumerable<Document>
             {
                 try
                 {
+                    var symbolContext = new RenameSymbolContext
+                    (
+                        ReplacementText: _replacementText,
+                        OriginalText: _originalText,
```

Comment:
Just to make sure I understand the PR sequencing here -- this seemed odd to me that the session still has a single _original/_replacmentText, but that work is coming next?

Reply:
Yes, that's something coming next. (See https://github.com/dotnet/roslyn/pull/63150)
I just don't want to touch `ConflictResover.Session` too much in this PR.